templates/emails/thankyou.hbs

```diff
@@ -27,6 +27,8 @@ Subject: Thank you for your contribution to {{collective.name}}
   <a href="https://opencollective.com/{{collective.slug}}">https://opencollective.com/{{collective.slug}}</a>, and share the link to help us spread the word!
 </p>
 
+{{{customMessage}}}
```

Comment:
What if there is none?

Reply:
When there's is none (the user empties the custom email section in settings or it's a new collective and they haven't configured the custom email section), we don't see anything here, since this {{{customMessage}}} will be empty. Since everything in this template is html, there's no additional spaces or anything added.